pagopa/io-app
# fix(IT Wallet): [SIW-1772] Mixpanel properties and events tracking issues

## Short description
This pull request includes several changes to the analytics tracking. Several issues related to updating some properties and events on mixpanel have been resolved.

## List of changes proposed in this pull request
- Removed eID properties update on focus, now it updates only when an action (creation or revocation) is completed.
- Fixed a check issue in the tracking functions related to checking the presence of eID inside the store
- Added some actions inside the machine to handle the tracking properly
- Fixed an issue related to the async flow tracking (tracked multiple times)
- Fixed an issue with the data share tracking (tracked multiple times)
- Added eid bottom sheet tracking

## How to test
Obtain a WI, add all the credentials, check on mixpanel that the various properties are updated correctly, remove the documents, check on mixpanel, finally revoke the WI, and check on mixpanel that all the properties have been updated correctly.
